usage/downstream-parent/pom.xml

```diff
@@ -62,7 +62,6 @@
     <jersey.version>1.18.1</jersey.version>
     <httpclient.version>4.4.1</httpclient.version>
     <commons-lang3.version>3.1</commons-lang3.version>
-    <groovy.version>2.3.4</groovy.version> <!-- Version supported by https://github.com/groovy/groovy-eclipse/wiki/Groovy-Eclipse-2.9.0-Release-Notes -->
```

Comment:
Do we really want to remove this? With OSGi, obviously we don't care about this kind of thing. But without that, someone writing entities that will be dropped into Brooklyn should not add a different version of groovy-all from the one already in Brooklyn.